Update README (#7)
antmicro/surelog-uhdm-ibex-guide: README.rst

The resulting bitstream file will be located in the ``build/lowrisc_ibex_top_artya7_surelog_0.1/synth-vivado/lowrisc_ibex_top_artya7_surelog_0.1.bit`` file

Enable additional features
--------------------------

> Some uhdm-plugin features requires modified yosys version. This can be done applying ``uhdm.patch`` to yosys repository and rebuilding yosys-symbiflow-plugins without ``BUILD_UPSTREAM`` flag.

Reviewer: ```suggestion
Some uhdm-plugin features require a modified version of Yosys. To use them, apply ``uhdm.patch`` to the Yosys repository and rebuild ``yosys-symbiflow-plugins`` without the ``BUILD_UPSTREAM`` flag.
```

Author: Fixed.
